src/main/java/org/myrobotlab/service/abstracts/AbstractSpeechSynthesis.java

```diff
@@ -40,20 +41,33 @@
    * substitutions are phonetic substitutions for a specific instance of speech
    * synthesis service
    */
-  Map<String, String> substitutions = new HashMap<String, String>();
-  
+  transient protected Map<String, String> substitutions = new ConcurrentHashMap<String, String>();

```

Comment:
These changes are unrelated to the metadata/service type change. please revert for this PR.